examples/tutorials/analysis-1d/spectral_analysis.py

```diff
@@ -419,6 +419,22 @@
 flux_points.plot_ts_profiles(ax=ax, sed_type="e2dnde")
 plt.show()
 
+######################################################################
+# We can also plot the flux distribution with the spectral model overlaid:
+#
```

Comment:
I think it would be good to add a comment on the importance of choosing the same energy binning as for the flux points for integral fluxes.